### PR TITLE
All stores flag support

### PR DESCRIPTION
**Pull request by spbsoluble**

### CODE REVIEW

File: readme_source.md [hunk 1]
```diff
@@ -192,33 +225,92 @@ kfutil stores rot generate-template --type stores \
...
+  --client $(kfutil orchs list | jq -r ".[0].ClientMachine")
+```
+
+#### Add certificates to certificate stores
+
+For full documentation, see [stores inventory add](docs/kfutil_stores_inventory_add.md).
+
+```bash
+# Add 2 certs to 2 certificate stores
+kfutil stores inventory add \
```
Comment (joevanwanzeeleKF) [inline readme_source.md:290]: Should we mention that the order of the parameters is important?  It should be kind of obvious.. but maybe?  Just a thought.

**Review by joevanwanzeeleKF — APPROVED** [review]
This looks wonderful.

File: readme_source.md (hunk 1 above)
Comment (spbsoluble) [reply readme_source.md:290]: I don't think the order matters with flags I think the only time order matters is if there is an arg and some CLIs don't care, others requires all flags to be after all args and there might even be some that require all flags to be set before all args.

File: cmd/login.go [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+	"strings"
+	"syscall"
+
+	"github.com/spf13/cobra"
+)
+
+const DEFAULT_CONFIG_FILE_NAME = "command_config.json"
+
+// loginCmd represents the login command
+var loginCmd = &cobra.Command{
```
[the remark this answers is not in the document]
Comment (spbsoluble) [reply cmd/login.go:25]: Yep and now it actually works: https://github.com/Keyfactor/kfutil/pull/10/commits/842ed9dc0ffdf5d09d54cd9b7e035e3e5b1e08a0

Well at least on linux/macOS.

**Review by joevanwanzeeleKF — APPROVED** [review]

**Pull request merged**